Lib/test/test_email/test__header_value_parser.py

```diff
@@ -383,6 +383,14 @@ def test_get_unstructured_ew_without_trailing_whitespace(self):
             [errors.InvalidHeaderDefect],
             '')
 
+    def test_get_unstructured_without_trailing_whitespace_hang_case(self):
+        self._test_get_x(self._get_unst,
+            '=?utf-8?q?somevalue?=aa',
+            '=?utf-8?q?somevalue?=aa',
```

Comment:
This should idealy return `somevalueaa` with a header defect of missing required whitespace after the encoded word.

Reply:
Done